src/main/java/org/commonjava/util/jhttpc/model/SiteConfig.java

```diff
@@ -76,11 +76,13 @@
 
     private HttpClientContext clientContextPrototype;
 
+    private final boolean hostnameVerifier;
```

Comment:
can we rename to `hostnameVerified`? I think this might work better for method naming / self documentation.

Reply:
Sure. Updated.